docs/release-notes/self-hosted/self-hosted-v13.0.0.md

```diff
@@ -0,0 +1,146 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v13.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.6
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.7
+---
+
+# Self-hosted v13.0.0
+
+These release notes are for [Codacy Self-hosted v13.0.0](https://github.com/codacy/chart/releases/tag/13.0.0){: target="_blank"}, released on October 26, 2023.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v13.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v13.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:
+
+    -   [Codacy Analysis CLI 7.9.4](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.9.4)
+    -   [Codacy Coverage Reporter 13.13.8](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.13.8)
+
+## Support for ingress-nginx 4.8.3
+
+This version of Codacy Self-hosted adds support for [ingress-nginx 4.8.3](https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-4.8.3), which fixes the following vulnerabilities:
+
+-   [CVE-2022\-4886](https://github.com/kubernetes/ingress-nginx/issues/10570)
+-   [CVE-2023\-5043](https://github.com/kubernetes/ingress-nginx/issues/10571)
+-   [CVE-2023\-5044](https://github.com/kubernetes/ingress-nginx/issues/10572)
+
+!!! note
+    Although [ingress-nginx 4.8.3 officially supports](https://github.com/kubernetes/ingress-nginx/tree/helm-chart-4.8.3#supported-versions-table) Kubernetes **version 1.25** to **1.28**, we tested this version of Codacy Self-hosted using [ingress-nginx 4.8.3](https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-4.8.3) on all the [supported Kubernetes versions](https://docs.codacy.com/v13.0/chart/requirements/#kubernetes-or-microk8s-cluster-setup), **1.22.\*** to **1.26.\***. All tests were successful in accordance with our quality assurance standards.
+
+### Upgrading ingress-nginx to version 4.8.3
+
+!!! warning
+    -   If you applied to ingress-nginx **custom configurations** different from the Codacy's `values-nginx.yaml` file, follow the [ingress-nginx documentation on how to upgrade your installation](https://github.com/kubernetes/ingress-nginx/blob/main/docs/deploy/upgrade.md).
```

Comment:
@tfrgoncalves Is this the correct ingress-nginx upgrade link to point to?

Reply:
I haven't found any specific helpful link for upgrading. We could instead point to the chart release:
... follow [ingress-nginx documentation](https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx/4.8.3)

Reply:
```suggestion
    -   If you applied **custom configurations** to ingress-nginx different from the Codacy's `values-nginx.yaml` file, follow the [ingress-nginx documentation](https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx/4.8.3) to upgrade your installation.
```